test/ansible/run_test.yaml

```diff
@@ -148,6 +154,15 @@
   - name: Debug tests output
     debug: "msg={{ loadbalancerip.stdout }}"
 
+  - name: prepare infrastructure-operator on target
+    shell: |
+      envsubst < "{{ src_dir }}/test/e2e/manifests/infrastructure-operator/kustomization.yaml.tpl" > "{{ src_dir }}/test/e2e/manifests/infrastructure-operator/kustomization.yaml"
```

Comment:
We have recently added jinja2 templates in ansible tests, we should probably turn this template also in a jinjia template producing a kustomization.yaml